docs/UserGuide.md

```diff
@@ -134,8 +193,14 @@ In a selected *SECTION*,
 ## Command Summary
 {Give a 'cheat sheet' of commands here}
 
+###Planner Mode
+**Command** | **Format** | **Example**
+----------- | ---------- | -----------
+Add a task: `add` | add /tTASK /by[dd/MM/yyyy] [hhmm] | add /tcoding /by19-10-2020 1705
+
+### Notebook Mode
 **Command** | **Format** | **Example**
 ----------- | ---------- | -----------
-*TIMETABLE MODE* `add` | add /tTASK /by[dd/MM/yyyy] [hhmm] | add /tcoding /by19-10-2020 1705
-*NOTEBOOK MODE* `add` | 1) add /nNOTEBOOK 2) add /sSECTION 3) add /pPAGE; CONTENT | add /nCS2101
-*NOTEBOOK MODE* `select` | 1) select /nNOTEBOOK 2) select /sSECTION 3) select /pNUMBER | select /nCS2101
+Add: `add` | 1) add /nNOTEBOOK 2) add /sSECTION 3) add /pPAGE; CONTENT | add /nCS2101
+Select: `select` | 1) select /nNOTEBOOK 2) select /sSECTION 3) select /pNUMBER | select /nCS2101
+Delete: `delete` | 1) select /nNOTEBOOK /sSECTION /pNUMBER | select /nCS2113T /sW10 /p1
```

Comment:
oh yes separating the commands for the different modes seems like a good idea! 😀

Reply:
Totally agree!